src/Compilers/CSharp/Portable/Compiler/Instrumentation.cs

```diff
@@ -199,11 +199,52 @@ public override BoundNode Visit(BoundNode node)
                         }
                         break;
                     case BoundKind.LocalDeclaration:
```

Comment:
I've pushed a commit that suppresses instrumentation of local declarations with no initialization, to the nearest approximation practical before we switch to the new architecture.